pro/cypress/e2e/step-definitions/signupJourneyCreateOfferer.cy.ts

```diff
@@ -136,14 +105,27 @@ When('I validate the registration', () => {
 
 When('I add a new offerer', () => {
   cy.url().should('contain', '/parcours-inscription/structure/rattachement')
+  cy.intercept(
+    'GET',
+    'https://api-adresse.data.gouv.fr/search/?limit=5&q=*'
+  ).as('search5Address')
   cy.findByText('Ajouter une nouvelle structure').click()
+  cy.wait('@search5Address')
 })
 
 When('I fill identification step', () => {
```

Comment:
Je viens de remarquer que cette étape ainsi que `I add details to offerer` pourraient être fusionnées. Il y a une raison pour que ça ne soit pas le cas ?

Reply:
Ils font des choses différentes, mais aussi des choses en commun. Je propose de renommer l'action pour que ce soit plus clair (voir dernier commit). Bien vu en tout cas